feature-group-definitions/page-breaks.yml

```diff
@@ -0,0 +1,18 @@
+name: Page breaks
+description: The `break-after`, `break-before`, `break-inside` CSS properties (along with `page-break-` aliases) control where pages start and end. Also known as pagination or page breaking.
```

Comment:
Here, I suspect there's no syntax to point to for context. But if this is only for printing, perhaps mention that?